### ISSUE TITLE
Python logs out to Docker container

### ISSUE THREAD

**Issue by kylebyrne** [kastldratza/zoomrec#22]
Hey there, firstly thanks for the great container!
I'm having some troubles with getting useful output from the python script out to my calls to `docker logs`. 

In my output I get the logs from the entrypoint file and then an error which I believe comes from the `xfce4-terminal` call?

```
Failed to connect to session manager: Failed to connect to the session manager: SESSION_MANAGER environment variable not defined
```

After this I'm  not able to see any of the python output, is this because its running inside of the separate terminal? Do you know of anyway to pipe the output from python out to stdout on the container?

**Comment by kylebyrne** [comment]
Github, the perfect rubber duck.
Managed to figure this out, for anyone else that comes across, I got the python logging to output to a file rather than just to stdout inside the other process.
e.g. something like
```python
logging.basicConfig(
    filename=os.path.join(BASE_PATH, "output.log"),
    filemode="w",
    format='%(asctime)s %(levelname)s %(message)s', level=logging.INFO
)
```
Then in entrypoint.sh I added `ln -sf /proc/1/fd/1 "$HOME"/output.log` it turns out that because we're running a background process we can't just symlink `/dev/stdout` as some sources say ( Loads more detail [here](https://stackoverflow.com/questions/58716574/where-exactly-do-the-logs-of-kubernetes-pods-come-from-at-the-container-level/58718841#58718841) )
